serivce-chain-azure/output.tf

```diff
@@ -0,0 +1,13 @@
+output "en_public_ip" {
+  value = "${module.en_vm.*.en_public_ip_address}"
+}
+
+output "en_private_ip" {
+  description = "private ip addresses of the vm nics"
+  value       = "${module.en_vm.*.en_private_ip_address}"
+}
+
+output "scn_private_ip" {
+  description = "private ip addresses of the vm nics"
+  value       = "${module.scn_vm.*.scn_private_ip_address}"
+}
```

Comment:
If you use `vscode`, you can set the following options to automatically add newlines when saving and avoid these warnings:

```
    "files.insertFinalNewline": true,
    "files.trimTrailingWhitespace": true,
```

Reply:
OK